ctags/main/options.h

```diff
@@ -93,17 +133,63 @@ extern CONST_OPTION optionValues		Option;
 /*
 *   FUNCTION PROTOTYPES
 */
+extern void notice (const char *const format, ...) CTAGS_ATTR_PRINTF (1, 2);
 extern void verbose (const char *const format, ...) CTAGS_ATTR_PRINTF (1, 2);
+#define BEGIN_VERBOSE(VFP) do { if (Option.verbose) { \
+                                FILE* VFP = stderr
+#define END_VERBOSE()      } } while (0)
+
 extern void freeList (stringList** const pString);
 extern void setDefaultTagFileName (void);
+extern void checkOptions (void);
+extern bool filesRequired (void);
+extern void testEtagsInvocation (void);
+
+extern cookedArgs* cArgNewFromString (const char* string);
+extern cookedArgs* cArgNewFromArgv (char* const* const argv);
+extern cookedArgs* cArgNewFromFile (FILE* const fp);
+extern cookedArgs* cArgNewFromLineFile (FILE* const fp);
+extern void cArgDelete (cookedArgs* const current);
+extern bool cArgOff (cookedArgs* const current);
+extern bool cArgIsOption (cookedArgs* const current);
+extern const char* cArgItem (cookedArgs* const current);
+extern void cArgForth (cookedArgs* const current);
 
-extern bool isIncludeFile (const char *const fileName);
 extern bool isExcludedFile (const char* const name);
+extern bool isIncludeFile (const char *const fileName);
+/* GEANY DIFF */
+/* extern const ignoredTokenInfo * isIgnoreToken (const char *const name); */
 extern bool isIgnoreToken (const char *const name, bool *const pIgnoreParens, const char **const replacement);
```

Comment:
See my comment below. My preference would indeed be using the new implementation unless the loss of functionality is a big obstacle for someone.